packages/templates/src/raw/12-hours-in-barcelona/template.json

```diff
@@ -344,27 +271,16 @@
         },
         {
           "opacity": 100,
-          "flip": {
-            "vertical": false,
-            "horizontal": false
-          },
+          "flip": { "vertical": false, "horizontal": false },
           "rotationAngle": 0,
           "lockAspectRatio": true,
-          "backgroundColor": {
-            "color": {
-              "r": 255,
-              "g": 199,
-              "b": 0
-            }
-          },
+          "backgroundColor": { "color": { "r": 255, "g": 199, "b": 0 } },
```

Comment:
No idea why the formatting here suddenly changed... But Prettier thinks it's fine like this.